src/integration-test/groovy/org/broadinstitute/orsp/CrowdServiceSpec.groovy

```diff
@@ -32,49 +32,49 @@ class CrowdServiceSpec extends Specification {
     def cleanup() {
     }
 
-    void stubCrowdResource() {
```

Comment:
I suggest disabling these tests for this PR and removing it entirely for BTRX-600 (and replacing it with tests that cover the BQ implementation)